src/__tests__/unit/controllers/universal-router/test.tsx

```diff
@@ -126,7 +126,7 @@ describe('UniversalRouter', () => {
         .mockImplementation(() => mockHistory);
     });
 
-    it('should not respond to history changes', () => {
```

Comment:
This test was passing only because the history being called was legacy history. Listening to memory router on SSR is fine, there is no danger in that